src/utils.js

```diff
@@ -30,6 +33,7 @@ export function getDisplayName(WrappedComponent: string | Component): string {
 function getSizeName(size: number) {
   switch (size) {
     case 0:
+    case undefined:
```

Comment:
Undefined is "single" in terms of class names, should not be 0 unless you update the css